From 4a4daaa0e0ce3eb07f30655e0354cb0e2e8f1497 Mon Sep 17 00:00:00 2001
From: Joey Zheng <zhengyu@eswincomputing.com>
Date: Thu, 24 Apr 2025 16:53:28 +0800
Subject: [PATCH 486/652] feat:adapt to vpu7702 smmu cfg drv

Changelogs:
Update compatible name to match vpu7702 smmu cfg driver which is
updated for new bar mapping mechanism.

Signed-off-by: Joey Zheng <zhengyu@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-vpu7702.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-vpu7702.dts b/arch/riscv/boot/dts/eswin/eic7700-vpu7702.dts
index 4bdb2da8ba3fb..25c74eaa69605 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-vpu7702.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-vpu7702.dts
@@ -188,7 +188,7 @@ d0_ipcm:ipcm@0 {
 		};
 
 		dev_smmu_cfg: smmu_cfg@0 {
-			compatible = "riscv,dev_smmu_cfg";
+			compatible = "eswin,vpu7702_dev_smmu_cfg";
 			#size-cells = <2>;
 			dma-ranges = <0x0 0x20000000 0x0 0xc0000000 0x0 0x40000000>;
 			iommus = <&smmu0 WIN2030_SID_DEV_FOO_B>;
-- 
2.51.1

